### ISSUE TITLE
Mousecape won't show its windows

### ISSUE THREAD

**Issue by Pboy06** [alexzielenski/Mousecape#141]
Hello, the first time I opened Mousecape I was able to change the cursor and all things. When I went to open it the second time the application started but no windows exist. I have tried restarting my computer, re-installing the program, clearing the cache but nothing has worked. All other programs open and work fine.

Thanks

Patrick

**Comment by NordJan** [comment]
Hallo Pboy06, does the Mousecape application appear in the menubar after opening it? If so, go to Window (menu item first from the right) and select Mousecape (at the bottom). If not, is it really started?
Hope this helps. Let us know.
Cheers, Nord

**Comment by Pboy06** [comment]
@NordJan Im on mac, not windows

**Comment by NordJan** [comment]
Pboys06, I know, so am I and so is everyone else using Mousecape, because it is a Mac-only application.
Did you get any further with my suggestion?

**Comment by Pboy06** [comment]
@NordJan No, all the options are unable to be clicked.

**Comment by NordJan** [comment]
If you want any furhter assistance, it helps if you describe what does work and what doesn't, and maybe send a screenshot.
Meanwhile, I can think of: did you install the helper tool? (It is under the Mousecap menu item).

**Comment by Pboy06** [comment]
Yes I have installed the helper tool. 
I have no options in window.
![Screen Shot 2020-11-29 at 10 38 32 am](https://user-images.githubusercontent.com/75052711/100528274-101b2c00-322f-11eb-995e-bbcda6d33b1d.png)

**Comment by NordJan** [comment]
Hmmm, strange.
When you open the application, is it responsive? In other words, when you right-click the dock-icon, what is the last item, "quit" or "force quit"? Just trying to narrow down this issue.

**Comment by Pboy06** [comment]
@NordJan just "Quit"

**Comment by NordJan** [comment]
Ok, this looks like the application is running and responding.
In the application's menu, can you select File and click New Cape and send the screenshot of how it looks then?
And/or can you go to /Library/Application Support/Mousecape/capes and see how many .cape files are there. If there is more than zero, please double-click one and tell us what happened next.

**Comment by Pboy06** [comment]
@NordJan Nothing comes up, even using the command. When I open the .cape file nothing happens again. No windows, nothing.

**Comment by NordJan** [comment]
This is a tough one, but I am not giving up yet. Maybe the .cape is corrupt or ill-formatted.
Could you send the .cape you are trying to install here? So, attach the file to your next reply.
And could you try the .cape I am posting here? It is a .zip, so it must be uncompressed first, I think you know this drill.
[Solara.SkinPlant.cape.zip](https://github.com/alexzielenski/Mousecape/files/5615116/Solara.SkinPlant.cape.zip)

**Comment by bunnieboo** [comment]
Okay so I had a basically the same issue and here's what I did to FINALLY get it to work after days,

- install appdelete 
- drag the Mousecape app to appdelete *this will delete the application and all files associated with it*
- reinstall Mousecape and it'll open!
this is the ONLY solution that has worked for me

**Comment by michael12345678939** [comment]
i have this same issue it was working then this happend

**Comment by ademaslanisthebest** [comment]
> Okay so I had a basically the same issue and here's what I did to FINALLY get it to work after days,
> 
> * install appdelete
> * drag the Mousecape app to appdelete _this will delete the application and all files associated with it_
> * reinstall Mousecape and it'll open!
>   this is the ONLY solution that has worked for me

this works i have tried it and i can now access it perfectly. Hope i dont need to delete in the future

**Comment by angeloftheweb** [comment]
I tried doing this and it still didn’t load for me. Any more tips?

![Image](https://github.com/user-attachments/assets/a6deff0b-ab5b-46df-a464-4868040335bf)

![Image](https://github.com/user-attachments/assets/c0b27852-6d74-416d-a00e-b899d0512e27) I downloaded the app cleaner (idk if this is the correct one) but I put mouse cape there then redownloaded it and nothing still comes up

**Comment by NordJan** [comment]
Dear angeloftheweb, before you can load any cursor sets, you have to open (or import) a .cape file. I don't see any in the MouseCape window.
So, first download a .cape file, open it from within MouseCape, it should appear as a row of different cursors, then choose (double-click) this set. Let us know how this works out for you.
Cheers, Nord

**Comment by angeloftheweb** [comment]
This is what comes up when I press for a new cape file![image](https://github.com/user-attachments/assets/f762f762-1917-4c8a-a44d-e980dcaabd18) not sure if this is the only thing that's supposed to come up but it won't let me type anything

**Comment by angeloftheweb** [comment]
also this is my first time using mousescape but do I download cape files by google searching what I want?

**Comment by NordJan** [comment]
> This is what comes up when I press for a new cape file![image](https://github.com/user-attachments/assets/f762f762-1917-4c8a-a44d-e980dcaabd18) not sure if this is the only thing that's supposed to come up but it won't let me type anything

Ah, it seems you are opening a new cape file and of course this is empty. Try to download the attached file (unzip to have the .cape file) and apply this, by importing it from within MouseCape(or double clicking the cape file). Let us know what happens then. Nord

[Solara.SkinPlant.cape.zip](https://github.com/user-attachments/files/23449396/Solara.SkinPlant.cape.zip)

**Comment by angeloftheweb** [comment]
I think I got it to the one I wanted, do you know a way I can do an animated one if I wanted to?

**Comment by NordJan** [comment]
Yes, you can not use .ani files from Windows, they are not supported. You have to make a vertical stack of pictures, see https://github.com/alexzielenski/Mousecape#how-do-animated-cursors-work

**Comment by angeloftheweb** [comment]
thanks so much!

**Comment by angeloftheweb** [comment]
One last question, is there a way to keep the mouse after you restart and then turn off / on your laptop bc everytime I do that the mouse isn't there and I have to reapply it by opening mousescape, is there a way to fix that or do I just have to keep reapplying?

**Comment by NordJan** [comment]
One last answer ;-).
Yes, but first a few questions: did you install the helper tool (it is under the Mousecape menu item)? What version of macOS are you using? What version of Mousecape do you have installed? Depending on your answers we will go further ahead.

**Comment by angeloftheweb** [comment]
Yes the helper tool is installed, I'm currently on MacOS Sequoia 15.7.2 and it says I have version 1813 for mousescape

**Comment by NordJan** [comment]
Ah, that is exactly  what I am using (and for me my favorite cursor set automagically re-applies after reboot).
Please look in the System Settings: under General and then Login Items & Extensions, scroll to Allow in the Background, Mousecape should be allowed (well, it is for me). Cheers, Nord

**Comment by NordJan** [comment]
@angeloftheweb I just noticed there are SD and HD .cape's. I did a test with a different cursor set, a HD one, and it didn't stick after reboot, so it might be dependent on that too. My favorite one, Solara, is SD.
Try a SD one and let us know what happens.

**Comment by angeloftheweb** [comment]
How do you switch it? I'm pretty sure the one I have is SD so I don't know why it's not sticking

**Comment by NordJan** [comment]
You need to have at least 2 .cape's to be able to switch. The SD or HD is at the right hand side of the name bar.
Can you upload the one you ate using, so we can test this out what it does in our set up?
Also, you can download "my" .cape and see what happens at your end.

**Comment by angeloftheweb** [comment]
this is the one I'm using ![image](https://github.com/user-attachments/assets/6f740851-fe68-4e2b-bbc4-03646717e230)

Then when I try downloading your cape ![image](https://github.com/user-attachments/assets/4781064c-4ba5-4829-8f6d-95f0eac95e3a) I then restarted my laptop to see if it would stay and for some reason neither one of them is staying unless I open mousescape ![image](https://github.com/user-attachments/assets/9e26ec4d-1c57-40f2-b859-7b7f3ec86197)

**Comment by angeloftheweb** [comment]
![image](https://github.com/user-attachments/assets/3c09bce4-7937-4a51-b755-8f989d2b616c) (this was when I restarted and the mouse didn't apply unless I opened mousescape)

**Comment by NordJan** [comment]
Hmm, I am running out of options. I tried a different SD .cape myself and it didn't stick after reboot. But Solara does, so I am confused. I must have done something in the past to make Solara reload automagically after reboot. There might be someone else to shed a new light on this.

**Comment by angeloftheweb** [comment]
I hope there is cause it's a bit annoying to have to reopen mousescape every single time to actually get the cursor so I hope there's a way, and I appreciate you because you've still been the upmost of help even if you aren't able to figure this out at the moment
